demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -2503,3 +2506,224 @@ def test_is_command_or_script_name_starts_with_digit_valid():
+            ),
+            {"valid_policy_1"},
+            0,
+            id="integration_with_empty_policy_list",
+        ),
+        pytest.param(
+            create_integration_object(
+                paths=["script.commands"],
+                values=[
+                    [
+                        {
+                            "name": "test-command",
+                            "compliantpolicies": [
+                                "invalid_policy_1",
+                                "invalid_policy_2",
+                            ],
+                        }
+                    ]
+                ],
+            ),
+            {"valid_policy_1", "valid_policy_2"},
+            2,
+            id="integration_with_multiple_invalid_policies",
+        ),
+        pytest.param(
+            create_integration_object(
+                paths=["script.commands"],
+                values=[
+                    [
+                        {"name": "command1", "compliantpolicies": ["valid_policy_1"]},
+                        {"name": "command2", "compliantpolicies": ["invalid_policy"]},
+                    ]
+                ],
+            ),
+            {"valid_policy_1", "valid_policy_2"},
+            1,
+            id="integration_with_multiple_commands_one_invalid_policy",
+        ),
+        pytest.param(
+            create_integration_object(
+                paths=["script.commands"],
+                values=[
+                    [
+                        {"name": "command1", "compliantpolicies": ["invalid_policy_1"]},
+                        {"name": "command2", "compliantpolicies": ["invalid_policy_2"]},
+                    ]
+                ],
+            ),
+            {"valid_policy_1", "valid_policy_2"},
+            2,
+            id="integration_with_multiple_commands_multiple_invalid_policies",
+        ),
+    ],
+)
+def test_integration_compliant_policy_name_validator(
+    content_item, policy_names, expected_failures
+):
+    """
+    Given
+    - An integration with commands containing compliant policies.
+    When
+    - Calling the IsValidCompliantPolicyNameValidator content_contains_invalid_compliant_policy_name function.
+    Then
+    - Make sure the expected number of failures are returned.
+
+    Test cases:
+    - An integration with valid policy name
+    - An integration with invalid policy name
+    - An integration with multiple valid policies
+    - An integration with a mix of valid and invalid policies
+    - An integration with empty policy list
+    - An integration with multiple invalid policies
+    - An integration with multiple commands where one has invalid policy
+    - An integration with multiple commands where multiple have invalid policies
+    """
+    results = IsValidCompliantPolicyNameValidator().content_contains_invalid_compliant_policy_name(
+        content_item, policy_names
+    )
+    assert len(results) == expected_failures
```

Comment:
same comments :)

Reply:
Done. The UTs no longer counting invalid policies but the actual content of the invalid policy names in assertion with the expected results.